### PR TITLE
Disable cranelift optimizations to avoid bugs

### LINKED ISSUE #404: [ICE] codegen error for while in switch
### Code

This is an upstream bug in cranelift: https://github.com/bytecodealliance/wasmtime/issues/1648

```c
void foo(void) {
  switch (1) {
    while(1) case 1:;
    case 0:;
  }
}
The application panicked (crashed).
Message:  assertion failed: !self.opcode().is_branch()
Location: /home/joshua/.local/lib/cargo/registry/src/github.com-1ecc6299db9ec823/cranelift-codegen-0.63.0/src/ir/instructions.rs:254
```

### Expected behavior

Should loop forever at runtime.

### PR DESCRIPTION
**Pull request by jyn514**
r? @pythondude325 

Closes #404

### CODE REVIEW

**Comment by jyn514** [comment]
I actually thought they were disabled already, it turns out that `speed` means the speed of the generated binary, not compilation speed.

File: src/ir/mod.rs [hunk 1]
```diff
@@ -51,10 +51,6 @@ pub(crate) fn get_isa(jit: bool) -> Box<dyn TargetIsa + 'static> {
     flags_builder
         .enable("enable_verifier")
         .expect("enable_verifier should be a valid option");
-    // minimal optimizations
-    flags_builder
-        .set("opt_level", "speed")
```
Comment (pythondude325) [inline src/ir/mod.rs:56]: Should it be set to "none" instead, or is none a well defined default?

Comment (jyn514) [reply src/ir/mod.rs:56]: 'well defined' is a little unclear in the context of cranelift ... it's just as likely for `none` to be removed as for the default to be changed in the next release. As long as we have the test suite I'll notice so I think it's ok. See also https://github.com/bytecodealliance/wasmtime/issues/1634

Comment (pythondude325) [reply src/ir/mod.rs:56]: Cranelift releases are fun.

**Comment by pythondude325** [comment]
I say you can go ahead and merge it.

**Pull request merged**